src/client/method-status.c

```diff
@@ -266,7 +273,21 @@ static int get_status_unit_on(Client *client, char *node_name, char *unit_name,
         return 0;
 }
 
-static int method_status_unit_on(Client *client, char *node_name, char **units, size_t units_count) {
+
+static int on_unit_status_changed(UNUSED sd_bus_message *m, void *userdata, UNUSED sd_bus_error *error) {
+        units_status *user_data = (units_status *) userdata;
+        printf("\033[2J");
+        printf("\033[%d;%dH", 0, 0);
```

Comment:
I think it would be great to move these two lines to a dedicated function `clear_screen()` in `method-status.c` and use it here as well as in other places - this way its more descriptive what happens (the escape codes are a bit tricky to read).